### ISSUE TITLE
Embedded HTML tags in Markdown

### ISSUE THREAD

**Issue by M1S2** [whistyun/MdXaml#38]
Is it possible to show some HTML tags that are inside the Markdown string using the MarkdownScrollViewer?
E.g.  
`<img src="https://github.com/M1S2/Ehrungsprogramm/raw/master/Screenshots/Screenshot_Start.png" width="45%"></img>`

Example Readme file showing an example for the embedded HTML tags in the Markdown string:
https://github.com/M1S2/Ehrungsprogramm

**Comment by whistyun** [comment]
Sorry for late update, I released v1.16.0.
It has plugin feature to enable html and svg support.

If you want to enable these support, please follow the steps below to set up your project

1. Add nupkg `MdXaml.Plugins` and `MdXaml.Html`, `MdXaml.Svg`
2. Edit App.xaml like the bellow sample
   ```xml
   <Application
                x:Class="*** CHANGE IT TO SUIT YOUR PROJECT ***"
                StartupUri="*** CHANGE IT TO SUIT YOUR PROJECT ***"
                xmlns="http://schemas.microsoft.com/winfx/2006/xaml/presentation"
                xmlns:x="http://schemas.microsoft.com/winfx/2006/xaml"
                xmlns:mdplug="clr-namespace:MdXaml.Plugins;assembly=MdXaml.Plugins"
                xmlns:mdhtml="clr-namespace:MdXaml.Html;assembly=MdXaml.Html"
                xmlns:mdsvg="clr-namespace:MdXaml.Svg;assembly=MdXaml.Svg">
       <Application.Resources>
           <mdplug:MdXamlPlugins x:Key="MdXamlPlugins">
               <mdhtml:HtmlPluginSetup/>
               <mdsvg:SvgPluginSetup/>
           </mdplug:MdXamlPlugins>
       </Application.Resources>
   </Application>
   ```

**Comment by M1S2** [comment]
Thank you for this feature. It's working great.
